### DISCUSSION TITLE
Is it possible to manage the (website|wiki|blog) entirely from the browser?

### DISCUSSION THREAD

**Discussion by munyard** [datenstrom/community#440] [Ask a question] [answered]
Is it possible to manage the (website|wiki|blog) entirely from the browser/edit screen?  

I've not found a way to make new folders from the browser/edit? is there a way to do this? 

When I create new documents they do not appear automatically anywhere in the menu however I discovered that if I install the Sitemap Extension I can find the files I've created.

So I thought it might be nice to have a Primary and Secondary Menu: 

The primary menu shows the Folders.
The secondary menu shows the Files in the folder.

**Comment by schulle4u** [comment, marked as answer]
Website management in Datenstrom Yellow primarily focusses on files and folders. Therefore website management from the browser is limited to creation/deletion/editing of pages. The edit extension will create pages in the current folder. So if you create a new page while on the homepage, it will be created in the `1-home` folder as well. But you can create new content folders from the browser simply by adding it to the end of an edit URL, e. g.: 

`https://example.com/edit/new-folder/`

The website will tell you that the page doesn't exist and offer you the possibility to create it if you are logged in. However there's no possibility to sort folders, this has to be done in the file system directly or with an online file manager. 

See also: [how to make a website](https://extensions.datenstrom.se/help/how-to-make-a-website).

HTH
Steffen

**Comment by IngoBelka** [comment]
I'm using for a couple of month [Tiny File Manager](https://tinyfilemanager.github.io/). It works very well and implements all I need for managing the site right in the browser. May be it's helpfully for others too.

Greetings Ingo

**Reply by GiovanniSalmeri** [reply]
@IngoBelka Thank you for signalling it! Just in order to understand your use case: do you use now exclusively Tiny File Manager to manage your site? do you see any advantages upon (for example) a simple FTP client?

**Comment by IngoBelka** [comment]
@GiovanniSalmeri We have a saying in Germany: There lead a lot of ways to Rome.
No, I do not use TFM exclusively. The advantages are most of the functionallity of a real file manager. In addition one can create zip files of an entire path. I use this like a "backup" before I make greater changes on system parts. You can unzip uploaded files, so you have faster upload and do the rest on the server. And TFM has a simple and an advanced editor included, what is a very nice feature to me. TFM is showing images, if you put the cursor over a file name.
And, all work is in the browser, no other application is needed. 
I suggest, to try it out. Installing is real easy.

**Reply by GiovanniSalmeri** [reply]
@IngoBelka Thank you very much for the explanation. Probably I feel less the need for such a tool because my FTP client *is* the File Manager: I use always Linux and the virtual file system allows me to use the File Manager on remote files exactly as if they were local files, for example I can edit them with the same editor I use for every other task (only I'm not allowed to compress or decompress). But I know that for example with Windows this is largely impossible (or unreliable: I have seen with my eyes incredible bugs), so a tool like TFM can be indeed very handy.

**Comment by IngoBelka** [comment]
Usally I have a local linux server for developement, with a normal root access.
But if I don't have direct access to the machine it is pretty helpfully to use a comfortable tool.
And for users having only shared webspace it might be a good solution, too.
